server/src/main/assets/css/partials/_main.scss

```diff
@@ -643,6 +647,55 @@ main {
               padding: 6px 12px;
             }
         }
+        .overlay {
+            position: fixed;
+            z-index: 10;
+            top: 0;
+            bottom: 0;
+            left: 0;
+            right: 0;
+            background: rgba(0, 0, 0, 0.7);
+            transition: opacity 500ms;
+            visibility: hidden;
+            opacity: 0;
+        }
+        .overlay:target {
```

Comment:
This is using the 'Pure-CSS lightbox' technique to create a modal overlay without any javascript:

https://developer.mozilla.org/en-US/docs/Web/CSS/:target#pure-css_lightbox

Reply:
Really nice too!